### PR TITLE
HW2: squashed hw1 commits into a single commit

### PR DESCRIPTION
**Pull request by Monirul1**
MD Monirul Islam
im2594

### CODE REVIEW

File: students/im2594.md [hunk 1]
```diff
@@ -12,4 +12,15 @@ Create a library which acts as a MongoDB wrapper in JavaScript to perform common
...
-I would like to contribute to open source projects or programming languages such as Protocol Buffer/gRPC and Golang/C++. I will look into the open issue section of these open source project and will attempt to pick an open issue and contribute into it.
+I would like to contribute to open source projects or programming languages such as Protocol Buffer/gRPC and Golang/C++. I will look into the open issue section of these open source project and will attempt to pick an open issue and contribute into it. 
+
+## Github Profile 
+GitHub: [link](https://github.com/monirul1)
+
+GitHub README: [link](https://github.com/monirul1/monirul1/blob/main/README.md)
+
+## Project Proposal
+- [mongo-wrapper](./projects/javascript/mongo-wrapper.md)
```
Comment (timkpaine) [inline students/im2594.md:23]: update link path and re-squash 😈

File: students/im2594.md [hunk 2]
```diff
@@ -12,4 +12,15 @@ Create a library which acts as a MongoDB wrapper in JavaScript to perform common
...
+
+## Project Contributions
+- [buf](./projects/buf/buf-issues.md)
```
Comment (timkpaine) [inline students/im2594.md:26]: update link path and re-squash 😈

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: students/im2594.md (hunk 1 above)
Comment (Monirul1) [reply students/im2594.md:23]: Hi Professor @timkpaine , I'm sorry I didn't understand, what do I need to update? this `./projects/javascript/mongo-wrapper.md` seem to be a right path as I want my read.md file to be under `./projects/javascript`?

File: students/im2594.md (hunk 2 above)
Comment (Monirul1) [reply students/im2594.md:26]: Got it, I will update this path `./projects/buf/buf-issues.md` to be `./projects/golang/buf-issues.md`

**Comment by Monirul1** [comment]
Hi Professor @timkpaine after I had squashed and committed looks a lot of other commits for included as part of my change, is that fine? thanks

**Pull request closed without merging**